gm4/plugins/manifest.py

```diff
@@ -61,7 +61,7 @@ class ManifestModuleModel(BaseModel):
 	name: str
 	version: str
 	hash: str
-	video_link: str = ""
+	video_link: Optional[str]
```

Comment:
Don't change the manifest format, keep this as an empty string, as the website may rely on it